### ISSUE TITLE
Problem with "UPLOAD" form

### ISSUE THREAD

**Issue by PDLee87** [jadchaar/sec-edgar-downloader#106]
Hi, according to the supported SEC filing type, the downloader is able to get "UPLOAD" However, when I tried to download the forms for Microsoft for example as below:

from sec_edgar_downloader import Downloader
dl = Downloader("directory folder")
dl.get("UPLOAD", "MSFT")

it does not download anything. Can you please let me know why? Thank you!

**Comment by monashjg** [comment]
I have the same problem. It seems the downloader does not cover all the filings. Any suggestions?

**Comment by jadchaar** [comment]
Hi @PDLee87 and @monashjg this has been fixed in v5.0 of this package. I was able to verify the UPLOAD files coming through now:
```
from sec_edgar_downloader import Downloader

downloader = Downloader("Foo Bar", "foobar@gmail.com")
downloaded = downloader.get("UPLOAD", "MSFT", limit=1)
print(downloaded) # outputs 1
```

Validated that UPLOAD file was downloaded successfully.
